monkey/infection_monkey/network/network_scanner.py

```diff
@@ -68,49 +90,44 @@ def get_victim_machines(self, max_find=5, stop_callback=None):
         :param stop_callback: A callback to check at any point if we should stop scanning
         :return: yields a sequence of VictimHost instances
         """
+        # We currently use the ITERATION_BLOCK_SIZE as the pool size, however, this may not be the best decision
```

Comment:
If this is the only place ITERATION_BLOCK_SIZE is used, then maybe we should refactor it to PROCESS_POOL_SIZE or PROCESS_COUNT?